src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.date.toLocaleTimeString());
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props): void {
+    const { clockName } = this.props;
+    const { clockName: prevName } = prevProps;
+
+    if (prevName !== clockName) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevName} to ${clockName}`);
```

Comment:
Nice job!
Please change *console.debug* to *console.info* 
See more information [link](https://developer.mozilla.org/ru/docs/Web/API/Console/debug)

Reply:
Thanks for the comment;)